<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CFunctional</class>
  <widget class="QDialog" name="CFunctional">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>213</width>
        <height>127</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle">
      <string>Distribution</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <item>
            <widget class="QLabel" name="label">
              <property name="text">
                <string>Distribution</string>
              </property>
              <property name="toolTip">
                <string>Function type</string>
              </property>
              
              <property name="whatsThis">
                <string>Distribution function type</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBox">
              <property name="toolTip">
                <string>Function type</string>
              </property>
              
              <property name="whatsThis">
                <string>Distribution function type</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
            <widget class="QLabel" name="labelParam1">
              <property name="text">
                <string>Param1</string>
              </property>
              <property name="toolTip">
                <string>Parameters of the functional distribution</string>
              </property>
              
              <property name="whatsThis">
                <string>Parameters of the selected functional distribution</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditParam1">
              <property name="toolTip">
                <string>Parameters of the functional distribution</string>
              </property>
              
              <property name="whatsThis">
                <string>Parameters of the selected functional distribution</string>
              </property>
            </widget>
          </item>
          <item row="0" column="2">
            <widget class="QLabel" name="labelUnits1">
              <property name="text">
                <string>[-]</string>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="labelParam2">
              <property name="text">
                <string>Param2</string>
              </property>
              <property name="toolTip">
                <string>Parameters of the functional distribution</string>
              </property>
              
              <property name="whatsThis">
                <string>Parameters of the selected functional distribution</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditParam2">
              <property name="toolTip">
                <string>Parameters of the functional distribution</string>
              </property>
              
              <property name="whatsThis">
                <string>Parameters of the selected functional distribution</string>
              </property>
            </widget>
          </item>
          <item row="1" column="2">
            <widget class="QLabel" name="labelUnits2">
              <property name="text">
                <string>[-]</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
            <widget class="QPushButton" name="pushButtonOk">
              <property name="text">
                <string>OK</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pushButtonCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>comboBox</tabstop>
    <tabstop>lineEditParam1</tabstop>
    <tabstop>lineEditParam2</tabstop>
    <tabstop>pushButtonOk</tabstop>
    <tabstop>pushButtonCancel</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>